_includes/v22.2/known-limitations/cdc-transformations.md

```diff
@@ -0,0 +1,9 @@
+- It is necessary to pass the [`schema_change_policy='stop'`](create-changefeed.html#schema-policy) option in the changefeed creation statement when using the {% if page.name == "cdc-transformations.md" %} CDC transformations {% else %} [CDC transformations](cdc-transformations.html){% endif %} format. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/85143).
```

Comment:
nit: Consider starting this sentence with `Currently, it is necessary ...` to indicate that an active PR to address this exists. Although ... that PR is closed!